fvm/src/machine/default.rs

```diff
@@ -59,7 +59,7 @@ where
     /// * `externs`: Client-provided ["external"][`Externs`] methods for accessing chain state.
     pub fn new(context: &MachineContext, blockstore: B, externs: E) -> anyhow::Result<Self> {
         const SUPPORTED_VERSIONS: RangeInclusive<NetworkVersion> =
-            NetworkVersion::V21..=NetworkVersion::V21;
+            NetworkVersion::V22..=NetworkVersion::V22;
```

Comment:
There are existing tests which still are on V21 that is causing CI failures

Reply:
Looking at this again, should we just not add nv22 to the supported version, like is done in https://github.com/filecoin-project/ref-fvm/pull/1846